java/client/src/main/java/glide/api/commands/ConnectionManagementClusterCommands.java

```diff
@@ -6,7 +6,7 @@
 import java.util.concurrent.CompletableFuture;
 
 /**
- * Supports commands for the "Connection Management" group for cluster clients.
+ * Supports commands for the "Connection Management" group for cluster client.
```

Comment:
Technically, would be a bit more grammatically correct to say "for **a** cluster client". Applies throughout.

Reply:
Fixed